README.md

```diff
@@ -1,5 +1,36 @@
 # moxiedoc
 
+### Introduction
```

Comment:
h1 -> h3 headings are not semantically correct. It looks like these should be h2 (`##`) headings instead.